packages/eslint-config-base/style.js

```diff
@@ -216,7 +216,8 @@ module.exports = {
     // Very long lines of code in any language can be difficult to read. In order to aid in
     // readability and maintainability many coders have developed a convention to limit lines of
     // code to a certain number of characters.
-    'max-len': ['warn', 100, 2],
+    'max-len': ['error',
```

Comment:
```suggestion
    'max-len': ['warn',
```

Should be a `warn` instead of `error`. Also we might consider adding `ignoreComments: true`, even tho it kinda break the whole idea of this rule 😅 

https://eslint.org/docs/rules/max-len

Reply:
Agree, let's use `warn`. I am willing to contemplate the idea of allowing `ignoreUrls` option but nothing more. 🤷‍♂️ 😇